packit_service/service/events.py

```diff
@@ -278,8 +280,10 @@ def get_dict(self) -> dict:
         return result
 
     def get_package_config(self) -> Optional[PackageConfig]:
-        if not self.base_ref:
-            self.base_ref = self.get_project().get_pr_info(self.issue_id).source_branch
+        releases = self.get_project().get_releases()
```

Comment:
I don't follow, what's wrong with master?

Reply:
Nothing, master is now default value of `base_ref`. Problem with the previous solution was that `get_pr_info(self.issue_id)` tracebacks because GitHub API cannot find PR with issue ID and issues have no ref.